languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -37745,6 +37745,93 @@ though, and has since been slightly modified)-->
                 <example correction=", il">Dehors<marker> il</marker> fait beau.</example>
                 <example><marker>Combien il</marker> en a pris ?</example>
             </rule>
+            <rule default="temp_off">
```

Comment:
Just to resume today's call:

- [ ] please create a rulegroup named `VIRGULE_DEBUT_DE_PHRASE_2` with these rules.
→ this allows us to look at the statistics separately. Later, we will merge it with the original rule.

- [ ] please set the rulegroup `temp_off` and erase the `default="temp_off"` from the subrules

- [ ] please remove the FPs, e.g. `!`, `que` etc.

… and let me know in a comment when you're done, so we get this rule rolling 😄